app/lib/frontend/handlers/pubapi.dart

```diff
@@ -45,7 +45,7 @@ class PubApi {
 
   /// Downloading package.
   /// https://github.com/dart-lang/pub/blob/master/doc/repository-spec-v2.md#download-a-specific-version-of-a-package
-  @EndPoint.get('/api/packages/<package>/versions/<version>.tar.gz')
+  @EndPoint.get('/api/packages/<package>/versions/<version>/archive.tar.gz')
   @EndPoint.get('/packages/<package>/versions/<version>.tar.gz')
```

Comment:
Can we add a comment here documenting that `/packages/<package>/versions/<version>.tar.gz` is hardcoded into the clients, so while this is deprecated we need to support it indefinitely?